pkg/steps/multi_stage.go

```diff
@@ -472,6 +473,14 @@ func (s *multiStageTestStep) generatePods(steps []api.LiteralTestStep, env []cor
 			errs = append(errs, err)
 			continue
 		}
+		allResources := &resources
+		if !resources.Requests.Name(api.ShmResource, resource.BinarySI).IsZero() {
+			// If shm is in Limits it must also be in Requests
+			allResources = resources.DeepCopy()
+			logrus.Info("removing shm from resources for container")
+			delete(resources.Requests, api.ShmResource)
+			delete(resources.Limits, api.ShmResource)
```

Comment:
Why do we delete them here? Do we want this on the container?

```
      resources:
        limits:
          ci-operator.openshift.io/shm: 2G
        requests:
          ci-operator.openshift.io/shm: 2G
```

Reply:
No, we can't have it in the container unless we actually "advertise" it as an extended resource. See this slack thread for a deeper explanation: https://coreos.slack.com/archives/GB7NB0CUC/p1635958978160200.